weave

```diff
@@ -952,17 +996,33 @@ attach() {
         connect_container_to_bridge $CONTAINER_IFNAME || return 1
     fi
 
+    if [[ -n "$AWSVPC" && $(echo "$@" | wc -w) -gt 1 ]]; then
+        echo "AWSVPC is enabled, therefore only one address can be attached to" >&2
```

Comment:
Yep, this is due to a way we detect subnet CIDR (`CIDR` at line 1009). Anyway, fixing it and getting rid of the limitation.